data/data-pipeline/data_pipeline/etl/score/tests/test_score_post.py

```diff
@@ -89,21 +88,21 @@ def test_create_tile_data(etl, score_data_expected, tile_data_expected):
     )
 
 
-def test_create_downloadable_data(
-    etl, score_data_expected, downloadable_data_expected
-):
-    output_downloadable_df_actual = etl._create_downloadable_data(
-        score_data_expected
-    )
-    pdt.assert_frame_equal(
-        output_downloadable_df_actual,
-        downloadable_data_expected,
-    )
+# def test_create_downloadable_data(
```

Comment:
what's going on here?

Reply:
See [comment here](https://github.com/usds/justice40-tool/pull/1252#pullrequestreview-886465283).

Maybe instead of commenting out, we should just take these tests out.